### PR TITLE
KEYCLOAK-10092 - Open House dummy bug

### PR DESCRIPTION
**Pull request by drichtarik**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <version>29</version>
     </parent>
 
-    <name>Keycloak</name>
+    <name>Keycloak - Open Hous</name>
```
Comment (mposolda) [inline pom.xml]: This looks like a typo "Hous" . Could you please fix it?

**Review by mposolda — COMMENTED** [review]
PR looks good. Just a minor typoe found.

File: pom.xml (hunk 1 above)
Comment (drichtarik) [reply pom.xml]: Fixed

**Review by mposolda — APPROVED** [review]
@drichtarik Thanks!

**Pull request merged**